### PR TITLE
Unit test for members

### PR DESCRIPTION
**Pull request by itemniner**
## Type of change made:
- [ ] **Bugfix** (non-breaking change which fixes an issue)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Refactoring** (improves existing functionality)
- [x] **Testing** (tests existing code or improves existing tests)
- [ ] **Styling** (no functionality changed, just visual improvements)
## Description:
- Test initialize method for members index and testing if object can be created with Propublica data with attributes in the correct format.
- Test initialize method for members show and test object can be created with Propublica data with attributes in the correct format.
## What functionality does this accomplish?
- Testing
closes # | does not close anything
## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

### CODE REVIEW

File: test/test_member_index.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import json
+from app import app
+from classes.member_index import MemberIndex
+
+test_browser = app.test_client()
```
Comment (SMJ289) [inline test/test_member_index.py:5]: I don't think we need this line for this test.

File: test/test_member_index.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  member = MemberIndex(data, state)
+
+  assert member.id == 'D000197'
+  assert member.image == 'https://theunitedstates.io/images/congress/original/D000197.jpg'
+  assert member.first_name == 'Diana'
+  assert member.last_name == 'DeGette'
+  assert member.role == 'Representative'
+  assert member.party == 'Democrat'
+  assert member.state == 'Colorado'
```
Comment (jklich151) [inline test/test_member_index.py:38]: Add test coverage for the api call of member index. Made test coverage % higher.

File: test/test_member_index.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import json
```
Comment (SMJ289) [inline test/test_member_index.py:1]: Or this line.

File: test/test_member_show.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
+import json
+from app import app
+from classes.member_show import MemberShow
+
+test_browser = app.test_client()
```
Comment (SMJ289) [inline test/test_member_show.py:5]: And here.

File: test/test_member_show.py [hunk 5]
```diff
@@ -0,0 +1,71 @@
+import json
```
Comment (SMJ289) [inline test/test_member_show.py:1]: Also, here.

File: test/test_member_index.py (hunk 2 above)
Comment (itemniner) [reply test/test_member_index.py:35]: okay we will make sure this works with test coverage

**Pull request merged**